docs/usage.md

```diff
@@ -1,261 +1,324 @@
-```
+### Digital normalization
 
-### Updating the pipeline
+Metatdenovo can perform "digital normalization" of the reads before the assembly.
+This will reduce coverage of highly abundant sequences and remove sequences that are below a threshold, and can be useful if the data set is too large to assemble but also potentially improve an assembly.
+N.B. the digital normalization is done only for the assembly and the full set of sequences will be used for quantification.
+To turn on digital normalization, use the `--bbnorm` parameter and, if required, adjust the `--bbnorm_target` and `--bbnorm_min` parameters.
 
-When you run the above command, Nextflow automatically pulls the pipeline code from GitHub and stores it as a cached version. When running the pipeline after this, it will always use the cached version if available - even if the pipeline has been updated since. To make sure that you're running the latest version of the pipeline, make sure that you regularly update the cached version of the pipeline:
+> Please, check the [bbnorm](https://jgi.doe.gov/data-and-tools/software-tools/bbtools/bb-tools-user-guide/bbnorm-guide/) documentation for further information about these programs and how digital normalization works. Remember to check [Parameters](https://nf-co.re/metatdenovo/parameters) page for the all options that can be used for this step.
 
-```console
-nextflow pull nf-core/metatdenovo
-```
+### Assembler options
 
-### Reproducibility
+By default, the pipeline uses Megahit (`--assembler megahit`) to assemble the cleaned and trimmed reads to create the reference contigs.
+Megahit is fast and it does not require a lot of memory to run, making it ideal for large sets of samples.
+The workflow also supports RNAspades, (`--assembler rnaspades` ) as an alternative.
 
-It is a good idea to specify a pipeline version when running the pipeline on your data. This ensures that a specific version of the pipeline code and software are used when you run your pipeline. If you keep using the same tag, you'll be running the same version of the pipeline, even if there have been changes to the code since.
+You can also choose to input contigs from an assembly that you made outside the pipeline using the `--assembly file.fna` (where `file.fna` is the name of a fasta file with contigs) option.
 
-First, go to the [nf-core/metatdenovo releases page](https://github.com/nf-core/metatdenovo/releases) and find the latest version number - numeric only (eg. `1.3.1`). Then specify this when running the pipeline with `-r` (one hyphen) - eg. `-r 1.3.1`.
+### ORF caller options
 
-This version number will be logged in reports when you run the pipeline, so that you'll know what you used when you look back in the future.
+By default, the pipeline uses prodigal (`--orf_caller prodigal` ) to call genes/ORFs from the assembly.
+This is suitable for prokaryotes, as is the Prokka alternative (`--orf_caller prokka`).
```

Comment:
Hasn't Prokka been succeeded by Bakta now?

Reply:
I didn't hear about it and I was just looking at their documentation. I know prokka can handle metatranscriptome and it looks like bakta is made primarily for MAGs.